js/main.js

```diff
@@ -0,0 +1,48 @@
+const addBooks = document.querySelector('#add');
+const bookName = document.getElementById('title');
+const authorName = document.getElementById('author');
+
+let bookList = [];
+
+function createBook(bookElement) {
+  document.querySelector('#book-list').innerHTML = '';
+  bookElement.forEach((e) => {
+    const element = document.createElement('books');
+
+    element.innerHTML += ` <li>
+        <p>${e.bookNames}</p>
+        <p>${e.authorNames}</p>
+        <input type="button" value="remove" onclick="removeBook(this.id)" id="${bookElement.length}">
+        <hr>
+    </li>`;
+    document.querySelector('#book-list').appendChild(element);
+  });
+  localStorage.clear();
+  localStorage.setItem('books', JSON.stringify(bookElement));
+}
+
+window.addEventListener('load', () => {
+  bookList = JSON.parse(localStorage.getItem('books'));
+
+  createBook(bookList);
+});
```

Comment:
- [ ] Now remove this reassignment of the bookList variable
```suggestion
window.addEventListener('load', () => {
  createBook(bookList);
});
```

Reply:
This issue is well resolved! ✔️